app/presenters/return-requirements/site-description.presenter.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+/**
+ * Formats data for the `/return-requirements/{sessionId}/site-description` page
+ * @module SiteDescriptionPresenter
+*/
+
+/**
+ * Formats data for the `/return-requirements/{sessionId}/site-description` page.
+ *
+ * @param {module:SessionModel} session - The returns requirements session instance.
+ * @param {Object} [payload] - The payload from the request.
+ *
+ * @returns {Object} - The data formatted for the view template.
```

Comment:
Remove the full stop off the end of these @param lines. [Take a look at the coding conventions and you'll see a good example of how a comment should look ](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#jsdoc-comments)